pages/_app.js

```diff
@@ -20,6 +20,12 @@ export default function App({ Component, pageProps }) {
         donations={donations}
         createDonation={createDonation}
       />
+      <Head>
+        <title>Essensretter</title>
+        <meta name="description" content="Generated by create next app" />
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+        <link rel="icon" href="/favicon.ico" />
+      </Head>
```

Comment:
normally its better to put the Head component first, since it is more common (it mimics the HTML file structure)

Reply:
ok thanks - i changed it :)